src/test/java/org/cyclonedx/maven/BaseCycloneDxMojoTest.java

```diff
@@ -0,0 +1,132 @@
+        developer.setName("Developer");
+        developers.add(developer);
+        developer = new Developer();
+        developer.setEmail("Developer@foo.com");
+        developers.add(developer);
+        developer = new Developer();
+        developer.setOrganization("My Organization");
+        developers.add(developer);
+        developer = new Developer();
+        developer.setOrganizationUrl("http://foo.com");
+        developers.add(developer);
+        List<OrganizationalContact> listOfAuthors = mojo.createListOfAuthors(manufacturer, developers);
+        assertNotNull(listOfAuthors);
+        assertEquals(4, listOfAuthors.size());
+        assertEquals("Developer", listOfAuthors.get(0).getName());
+    }
+
+    @Test
+    @DisplayName("Verify addContacts")
+    void addContacts() {
+        BaseCycloneDxMojoImpl mojo = new BaseCycloneDxMojoImpl();
+        OrganizationalEntity manufacturer = new OrganizationalEntity();
+        List<Developer > developers = new ArrayList<>();
+        mojo.addContacts(manufacturer, developers);
+        assertNotNull(manufacturer.getContacts());
+        assertTrue(manufacturer.getContacts().isEmpty());
+    }
+
+    @Test
+    @DisplayName("Verify addUrl")
+    void addUrl() {
+        BaseCycloneDxMojoImpl mojo = new BaseCycloneDxMojoImpl();
+        OrganizationalEntity manufacturer = new OrganizationalEntity();
+        assertNull(manufacturer.getUrls());
+        mojo.addUrl(manufacturer, "http://foo.com");
+        assertNotNull(manufacturer.getUrls());
+        assertEquals(1, manufacturer.getUrls().size());
+        mojo.addUrl(manufacturer, "http://example.com");
+        assertEquals(2, manufacturer.getUrls().size());
+    }
+
+
+    @Test
+    @DisplayName("Verify that check of String content works")
+    void isNotNullOrEmpty() {
+        BaseCycloneDxMojoImpl mojo = new BaseCycloneDxMojoImpl();
+        String value = null;
+        assertFalse(mojo.isNotNullOrEmpty(value));
+        value = "";
+        assertFalse(mojo.isNotNullOrEmpty(value));
+        value = "null";
+        assertTrue(mojo.isNotNullOrEmpty(value));
+    }
+
+
+
+    /**
+     * Inject a parameter value to a superclass (even private parameters).
+     * <p>example:</p>
+     * <br>{@code class B extends A;}
+     * <br>{@code class A  {  private Type a; } }
+     * <br>
+     * <br>{@code setParentParameter(new B(), "a", new Type()); }
+     *
+     * @param cc The class instance
+     * @param fieldName The field name
+     * @param value The value
+     * @throws NoSuchFieldException If the field does not exist
+     * @throws IllegalAccessException If the value is not able to be modified
+     */
+    public static void setParentParameter(Object cc, String fieldName, Object value)
+        throws NoSuchFieldException, IllegalAccessException {
+        Field field = cc.getClass().getSuperclass().getDeclaredField(fieldName);
+        field.setAccessible(true);
+        field.set(cc, value);
+    }
```

Comment:
Is this used anywhere?

Reply:
Sorry forgot to remove from the first PR, not used any more. It was used to set the private variable `manufacturer` before.